package.json

```diff
@@ -117,6 +118,7 @@
     "css-loader": "^6.7.1",
     "file-loader": "^6.2.0",
     "mocha": "^10.1.0",
+    "node-fetch": "2",
```

Comment:
node-fetch 2 is used because node-fetch 3 requires ESM support and it is not working with the way I call the script (in package;json using node) but I do not know how to launch in a different way. Open to any other library that can download a file. This one seemed used a lot and is relatively easy to use.